### PR TITLE
Add header componentry

### PR DESCRIPTION
**Pull request by lal65**
# Description:
This is largely new work; some updates to existing work.

### Review environment Link
  https://developers.outreach.psu.edu/bleeding-edge-headers

### CODE REVIEW

File: packages/header/src/scss/styles.scss [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+      &:nth-child(3) {
+        left: calc(50% + #{_rem(38)});
+      }
+    }
+  }
+
+  &__inner {
+    max-width: _rem(122);
+    margin: 0 auto;
+    padding: 0 _rem(2);
```
[the remark this answers is not in the document]
Comment (lal65) [reply packages/header/src/scss/styles.scss:55]: Nope.

**Review by kjl16 — CHANGES_REQUESTED** [review]
I didn't find anything glaring, just a few minor notes.

File: packages/horizontal-menu/src/scss/styles.scss [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    flex-wrap: wrap;
+
+    &--2 {
+      background: var(--white);
+      flex-direction: column;
+      position: absolute;
+      top: 100%;
+      opacity: 0;
+      visibility: hidden;
+      box-shadow: _rem(.1) _rem(.3) _rem(.5) rgba($nittany-navy, .75);
```
[the remark this answers is not in the document]
Comment (lal65) [reply packages/horizontal-menu/src/scss/styles.scss]: Done (and added @supports b/c layer).

File: packages/drop-button/src/js/scripts.js [hunk 3]
```diff
@@ -1,8 +1,130 @@
...
+        }
+        else {
+          panel.style.setProperty('--drop-button--left', 'calc(-1 * (var(--drop-button-panel-width, 28rem) - ' + ((bounds.width + 35) / px_to_rems_conversion) + 'rem))');
+          panel.style.setProperty('--drop-button--right', (-35 / px_to_rems_conversion) + 'rem');
+        }
+      }
+    });
+  };
+
+  // Whenever the body changes size under 950px, be sure to revalidate all
```
[the remark this answers is not in the document]
Comment (lal65) [reply packages/drop-button/src/js/scripts.js]: Probably 800.  Updated the comments.

File: packages/more-menu/src/scss/styles.scss [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+    &:hover {
+      background: var(--pugh-blue);
+    }
+
+    &:focus-visible {
+      background: var(--pugh-blue);
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (lal65) [reply packages/more-menu/src/scss/styles.scss:153]: No changes needed here - I updated the example in pattern lab in a previous commit :)

File: packages/more-menu/more-menu.twig [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      {% else %}
+        {% include '@atoms/sprite/sprite.twig' with { name: 'fas-ellipsis-vertical' } only %}
+      {% endif %}
+    </div>
+    <span class="more-menu__toggle-label">{{ more_label|default('More') }}</span>
+  </div>
+{% endset %}
+
+{% set link_list %}
+  <ul class="more-menu__list">
```
[the remark this answers is not in the document]
Comment (lal65) [reply packages/more-menu/more-menu.twig]: Done

**Pull request merged**